starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort();
+    this.length += 1;
+  }
+  get(pos) {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.item[this.length - 1];
+    }
+  }
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(this.items);
+    }
+  }
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.min(this.items);
+    }
+  }
+  average() {
+    this.items.sum / this.length;
```

Comment:
I like what you tried to do here, but you want to call the function so you need to add () `this.items.sum() / this.length;`